src/__tests__/service-metrics.test.ts

```diff
@@ -4,6 +4,12 @@ describe('simple test of metrics', () => {
   beforeAll(async () => {
     ServiceMetrics.start()
   })
+  test('trace span', async() => {
```

Comment:
What does the test check? o_O If it is an intentional noop to check availability of methods, could you mind please add a comment explaining that? I am afraid some next person could remove this test.

Reply:
yeah, the metrics methods are all no-op unless an exporter is actually present.  added comment